remo/featuredrep/forms.py

```diff
@@ -13,7 +16,15 @@ class FeaturedRepForm(happyforms.ModelForm):
             userprofile__registration_complete=True,
             groups__name='Rep').order_by('first_name'))
 
+    def clean_created_on(self):
+        print(self.cleaned_data['created_on'])
+        print(now() + datetime.timedelta(days=30))
+        if self.cleaned_data['created_on'] > now() + datetime.timedelta(days=30):
+            raise ValidationError('The date cannot be over 30 days in the future.')
```

Comment:
minor: "cannot be more than 30 days in the future". I'm not sure if 30 is the right value here, is there a way to do a "proper" month?

Reply:
actually no because we are doing a sum of 30 to the date of today.

Reply:
Which does not work out for months with 30 days as well as February, correct? ;)

Reply:
right but I think that we need that feature now and later we can improve and find a better way.
In php is different the management of dates so you don't need to do sums or put days for this kind of stuff. In few words I don't know if there is a better way.